control/controlclient/netmap.go

```diff
@@ -24,13 +24,12 @@ import (
 type NetworkMap struct {
 	// Core networking
 
+	SelfNode   *tailcfg.Node
 	NodeKey    tailcfg.NodeKey
 	PrivateKey wgkey.Private
 	Expiry     time.Time
```

Comment:
We could probably remove some of these other fields on `NetworkMap` now that we have `SelfNode`